images/installer.yml.in

```diff
@@ -17,6 +19,7 @@ onboot:
       - CAP_NET_ADMIN
   - name: installer
     image: INSTALLER_TAG
+    command: ["/bin/sh", "-c", "/install </dev/console >/dev/console 2>&1"]
```

Comment:
Why is this necessary? I don't mind, but if something is incorrect about the installer, can we fix it there? As the `CMD` in `pkg/installer/Dockerfile`?

Reply:
Done.